libs/utils/utils.js

```diff
@@ -827,11 +827,13 @@ async function checkForPageMods() {
   const persMd = getMetadata('personalization');
   const promoMd = getMetadata('manifestnames');
   const targetMd = getMetadata('target');
+  const dynamicNavMd = getMetadata('dynamic-nav');
   let persManifests = [];
   const persEnabled = persMd && persMd !== 'off' && !offFlag('personalization');
   const targetEnabled = targetMd && targetMd !== 'off' && !offFlag('target') && !offFlag('martech');
   const promoEnabled = promoMd && promoMd !== 'off' && !offFlag('promo');
   const mepEnabled = persEnabled || targetEnabled || promoEnabled;
+  const dynamicNavEnabled = dynamicNavMd && (dynamicNavMd === 'entry' || dynamicNavMd === 'on');
```

Comment:
why not `const dynamicNavEnabled = ['entry', 'on'].includes(getMetadata('dynamic-nav'))`